cmd_bind.go

```diff
@@ -6,14 +6,17 @@ package main
 
 import (
 	"fmt"
+	"github.com/gonuts/commander"
```

Comment:
please keep the "stdlib" / "3rd-party" separation of imports.

https://godoc.org/golang.org/x/tools/cmd/goimports should do that for you.